src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/DateOnlyConverter.cs

```diff
@@ -19,6 +18,9 @@ public class DateOnlyConverter : TypeConverter
         /// object in the given source type to a <see cref='System.DateOnly'/>
         /// object using the specified context.
         /// </summary>
+        /// <param name="context">An <see cref="T:System.ComponentModel.ITypeDescriptorContext" /> that provides a format context.</param>
```

Comment:
Can we use `/// <inhertdoc />`? That way we don't need to copy these strings everywhere.

cc @carlossanlop

Reply:
This is not exact. It is mentioning the type `DateOnly` as example.

Reply:
The summary is mentioning the type `DateOnly` which cannot inherit. 

Can we inherit some tags and not the others?

Reply:
I don't think it's possible.

Reply:
According to http://tunnelvisionlabs.github.io/SHFB/docs-master/SandcastleBuilder/html/79897974-ffc9-4b84-91a5-e50c66a0221d.htm

> Using it in conjunction with other tags allows you to inherit common information such as value and parameter descriptions while overriding the inherited documentation for other tags such as <summary> and <remarks>.

Reply:
Nice! TIL

Reply:
Is that the tool our docs team uses though? Might be worth checking with them it's ok.

Reply:
Quick search, I am seeing we already using it https://github.com/dotnet/runtime/blob/5b0baa9d1c76926aafc75dc48cdb76a28c345ffb/src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs#L329

Reply:
never mind. I see in the doc you sent has the summary listed before the `inherit`.